src/App.tsx

```diff
@@ -1,33 +1,69 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import logo from './assets/logo.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
     <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
+      <div className="header">
+        <a href="https://market.watch" target="_blank">
+          <img src={logo} className="logo" alt="MarketWatch logo" />
         </a>
+        <div className="navbar">
```

Comment:
Can you refactor this and make the navbar into a component instead? It will make the code more readable

Reply:
Hi I have done so in a new commit. Please let me know if there is other stuff I have to do (like pull req again or something, im not too familiar with this)